src/components/colorbar/draw.js

```diff
@@ -10,6 +10,7 @@
 'use strict';
 
 var d3 = require('d3');
+var tc = require('tinycolor2');
```

Comment:
The `tc` ones are a color variable name not the `tinycolor` module! I think they're ok in that context of the `color` module.